### PR TITLE
Added a commented config useful when you use symlinks

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.3+ |
| Fixed tickets | #4232 |

### CODE REVIEW

File: cookbook/configuration/web_server_configuration.rst [hunk 1]
```diff
@@ -48,6 +48,12 @@ are:
             Allow from All
         </Directory>
 
+        # uncomment the following lines if you install assets as symlinks
+        # or run into problems when compiling LESS/Sass/CoffeScript assets
+        # <Directory /var/www/project>
```
Comment (cordoval) [inline cookbook/configuration/web_server_configuration.rst]: shouldn't you suggest this change in here too https://github.com/symfony/symfony-standard/blob/2.7/web/.htaccess ?

Comment (javiereguiluz) [inline cookbook/configuration/web_server_configuration.rst]: Good point! When/if this change is validated for the Symfony docs, I'll add it to the `.htaccess` file of the Standard Edition. Thanks.

**Comment by weaverryan** [comment]
:thumbsup: I'm closing TONS of issues thanks to you :)

File: cookbook/configuration/web_server_configuration.rst (hunk 1 above)
Comment (xabbuh) [inline cookbook/configuration/web_server_configuration.rst]: @javiereguiluz You are read to go. :)

**Pull request merged**